novawallet/Common/Substrate/Operations/Identity/IdentityProxyFactory.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 import Operation_iOS
 
-protocol IdentityProxyFactoryProtocol {
+protocol IdentityDelegatedAccountFactoryProtocol {
```

Comment:
It's not related to proxy delegated account but for identity proxy. So i would keep name the same